mh/session.py

```diff
@@ -53,7 +53,8 @@ def __init__(self):
         self.layer = 0
         self.binary_setting = b""
         self.search_payload = None
-        self.binaries = None
+        self.binaries = None  # redundant?
```

Comment:
Yes, this can be removed. I planned to store the "stats" in that var. But you have one for that and with a much better name